### DISCUSSION TITLE
Feature Request: Analog Trigger Scaling Integration w/ GC+

### DISCUSSION THREAD

**Discussion by StonedEdge** [Aurelio92/RVLoader#30] [Ideas]
Request to add a similar sub menu to the joystick calibration menu within RVloader for custom analog trigger setups in portable Wii’s. This will initially read the min and max analog values, allow the user to press their trigger down to the maximum range and then offset the mix/max to a 0-255 analog input range.

**Comment by GameCube4Life** [comment]
I love this idea!!
